lobbyheads-api/src/main/java/com/eternalcode/lobbyheads/head/event/HeadCreateEvent.java

```diff
@@ -0,0 +1,40 @@
+package com.eternalcode.lobbyheads.head.event;
+
+import java.util.UUID;
+import org.bukkit.Location;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * This event is called when a head is created.
+ **/
+public class HeadCreateEvent extends Event {
+
+    private static final HandlerList HANDLER_LIST = new HandlerList();
+
+    private final UUID playerUniqueId;
```

Comment:
I think this name is nonsense, I would just change it to uniqueId as it is in bukkit-api

Reply:
rollczi

Reply:
Pierwsze co myślałem przy review kiedy było tutaj "uniqueId", to że jest to uuid heada. I mówię to jako osoba która po prostu czyta nieznajomy mi kod. Moim zdaniem nazwy zmiennych powinny odrazu mówić co reprezentują, dla osób które nie znają implementacji całego projektu.
W sumie skąd mam wiedzieć czy to jest   uuid głowy, uuid playera, uuid samego eventu albo jakiekolwiek inne uuid związane z tym wydarzeniem. 

To że istnieje metoda Player#getUuid nie oznacza, że w każdym miejscu w kodzie uuid to reprezentacja playera.
Głównie dlatego zmiennych się nie nazywa "number", "tab", "array", "info", "data", "list" bo to (zazwyczaj) nic nie oznacza, po za typem zmiennej.

Gdybym był w klasie User/Player etc to bym użył takiej nazwy. Ale tutaj mamy już inny kontekst i te uuid nie należy bezpośrednio do eventu.

No i przepraszam za Polski ale to nie pierwsza rozmowa w tej sprawie i chciałabym zostać dobrze zrozumiany.

Reply:
A i jeśli mieliśmyby sugerować się bukkit-api to wychodzi na to, że te uuid to jest uuid eventu. 

`Typ`#getUniqueId w bukkit'owym kontekście oznacza, że te uuid należy do `Typ`, a nie do czegoś innego.

Co wydaje się być całkiem logiczne.